src/lib/genesis_constants/genesis_constants.ml

```diff
@@ -105,84 +93,62 @@ module Constraint_constants = struct
               completing the proofs.
         *)
 
-      [%%inject "work_delay", scan_state_work_delay]
-
-      [%%inject "block_window_duration_ms", block_window_duration]
-
-      [%%if scan_state_with_tps_goal]
-
-      [%%inject "tps_goal_x10", scan_state_tps_goal_x10]
-
-      let max_coinbases = 2
-
-      (* block_window_duration is in milliseconds, so divide by 1000 divide
-         by 10 again because we have tps * 10
-      *)
-      let max_user_commands_per_block =
-        tps_goal_x10 * block_window_duration_ms / (1000 * 10)
-
-      (** Log of the capacity of transactions per transition.
-            - 1 will only work if we don't have prover fees.
-            - 2 will work with prover fees, but not if we want a transaction
-              included in every block.
-            - At least 3 ensures a transaction per block and the staged-ledger
-              unit tests pass.
-        *)
       let transaction_capacity_log_2 =
-        1
-        + Core_kernel.Int.ceil_log2 (max_user_commands_per_block + max_coinbases)
-
-      [%%else]
-
-      [%%inject
-      "transaction_capacity_log_2", scan_state_transaction_capacity_log_2]
-
-      [%%endif]
-
-      [%%inject "supercharged_coinbase_factor", supercharged_coinbase_factor]
+        match
```

Comment:
implicitly in the config it seems like you have one of two disjoint cases:
```
scan_state_with_tps_goal && 
  is_some scan_state_tps_goal_x10 && 
  is_none scan_state_transaction_capacity_log_2 
```
or 

```
not scan_state_with_tps_goal && 
  is_none scan_state_tps_goal_x10 && 
  is_some scan_state_transaction_capacity_log_2 
```

Reply:
what if we remove `scan_state_with_tps_goal`, `scan_state_tps_goal_x10` and retain only `scan_state_transaction_capacity_log_2` ?

Reply:
you probably could, but my goal here was to do the smallest transformation and keep any config values code paths that were being used. IMO it would be easier to simplify the config which _is_ being used in a follow up -- i.e. taking a look at what is in node config and seeing if we still care about each field.

For example, there are some fields which only act as a flag to log a simple statement, and other kind of "do nothing" fields we can likely remove.